### DISCUSSION TITLE
Create Space from API

### DISCUSSION THREAD

**Discussion by dhrudevaliasolidatus** [GitbookIO/community#205] [Feature Requests]
### What's your feature/idea?

Out desired use case for GitBook, is that on certain releases of our product we create a new space in a collection named as the version number and push the documentation changes. Therefore in GitBook there would be a drop down of a list of version numbers that correlated to different versions of the documentation.

Currently through the API it doesn't seem possible to create a new space rather just update an existing space. Is there a recommended way to mimic the above behaviour?

### What's your desired outcome?

More API endpoints that can create spaces/collections, etc, so that GitBook tooling can be integrated with the release process of the software.

### What's the impact of this for you?

- [ ] Critical
- [X] High
- [ ] Medium
- [ ] Low

**Comment by addisonschultz** [comment]
Hey @dhrudevaliasolidatus! Thanks for the great idea. I think adding this to the API would be definitely helpful. I made a note of this for us to look into! I think it should be something quick, so hopefully we can pick it up soon.

Stay tuned!

**Reply by dhrudevaliasolidatus** [reply]
Thanks!

**Reply by taszap** [reply]
https://developer.gitbook.com/gitbook-api/reference/spaces#post-a-new-space-in-an-organization 
This feature has become available 😄
